src/components/Button/index.js

```diff
@@ -1,22 +1,45 @@
+import './Button.scss'
```

Comment:
Internal imports are normally after external imports - applies multiple times!

Reply:
Ah, I just had a machine decide the order these were included in. They're just sorted alphabetically. Will change this so all external imports occur first.

Reply:
@aldreth Updated these imports. They now always go:

* [external imports, auto sorted alphbetically]
* [line break]
* [internal imports, auto sorted alphbetically]